test/organizationMemberships.matchers.js

```diff
@@ -0,0 +1,62 @@
+'use strict';
+
+const { _, expect, definePattern } = require('./matchers');
+require('./patterns');
+
+definePattern(
+  'anOrganizationMembership',
+  {
+    uid: _.isUid,
+    createdAt: _.isISODateString,
+    organizationApikey: _.isOrganizationApikey,
+    memberUserDid: _.isDID,
+    admin: _.isUndefinedOr(_.isBoolean),
+    curator: _.isUndefinedOr(_.isBoolean),
+    updatedAt: _.isUndefinedOr(_.isISODateString),
+  }
+);
+
+definePattern(
+  'anOrganizationAdminMembership',
+  target => {
+    expect(target).to.be.anOrganizationMembership();
+    expect(target).to.matchPattern({ admin: true, '...': 1 });
+  }
+);
+
+definePattern(
+  'anOrganizationCuratorMembership',
+  target => {
+    expect(target).to.be.anOrganizationMembership();
+    expect(target).to.matchPattern({ curator: true, '...': 1 });
+  }
+);
+
+definePattern(
+  'anOrganizationMembershipBetween',
+  (target, { organizationApikey, memberUserDid }) => {
+    expect(target).to.be.anOrganizationMembership;
```

Comment:
this is not being called

Reply:
https://media.giphy.com/media/3o6wrDdjMdUlO8PovS/giphy.gif